data_store/transformation/towns_fund/tf_transform_r3.py

```diff
@@ -635,7 +670,7 @@ def extract_funding_data(df_input: pd.DataFrame, project_lookup: dict, reporting
         mask, ["Funding Source Name", "Funding Source Type"]
     ].values
 
-    df_funding.index = df_funding["original_index"]
+    df_funding.index = pd.Index(df_funding["original_index"])
```

Comment:
What's the diff between assigning to `df.index` and `df.columns`? we seem to be setting pd.Index on both?

Reply:
thanks for explaining :)